spec/requests/v1/users_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Users', type: :request do
+  let(:user) { build(:user) }
+
+  describe 'POST /api/v1/registration' do
+    context 'when valid request' do
+      before do
+        post '/api/v1/registration', params: { email: user.email, password: user.password }
```

Comment:
What about path `/api/v1/users/registration`? We'll get more details what this endpoint do.

Reply:
Done